inc/ocf_def.h

```diff
@@ -175,6 +175,8 @@ typedef enum {
 		/*!< Current cache mode of given cache instance */
 } ocf_cache_mode_t;
 
+#define OCF_SEQ_CUTOFF_MAX_STREAMS 16
```

Comment:
Why did you go down to only 16 streams? Is rbtree actually performing better than linear search at this element count?

Reply:
Increased to 256.